### PR TITLE
new video for amp added

### LINKED ISSUE #38: Insert new video for the abap-metric-provider
https://youtu.be/K-e9gE_x4oU

### PR DESCRIPTION
**Pull request by JohannesKonings**
closes #38

### CODE REVIEW

File: src/components/Video.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ export default class Video extends React.Component {
         return (<div className="container container--md align-center">
             <ReactPlayer
                 url="https://www.youtube.com/watch?v=XV5brOMpzJA" />
+            <p />
```
Comment (Goala) [inline src/components/Video.js:9]: is <p /> the common way to add spacing? 
You may use a spacer component or something like that

Comment (JohannesKonings) [reply src/components/Video.js:9]: It was the fastest way I found :) Do you have a recommendation?

Comment (Goala) [reply src/components/Video.js:9]: maybe a div with the css class 'spacer' and the css definition of course 
this makes something like this also possible 
`<div class=spacer>< /div>`
`<div class=spacer--20>< /div>`
`<div class=spacer--40>< /div>`
for different sizes 
but not really necessary now

Comment (JohannesKonings) [reply src/components/Video.js]: I changed it to <br />. Should be OK for the moment.

**Review by Goala — APPROVED** [review]
lgtm

**Pull request merged**